code/modules/mob/language/station_vr.dm

```diff
@@ -70,6 +70,18 @@
 	syllables = list("salve","sum","loqui","operatur","iusta","et","permittit","facere","effercio","pluribus","enim","hoc",
 	"mihi","wan","six","salve","tartu")
 	machine_understands = FALSE
+	
+/datum/language/bug
+    name = LANGUAGE_VESPINAE
```

Comment:
# WE REQUIRE MORE VESPENE GAS

Reply:
Fun Fact: That name comes from "Vespid" which is the taxonomic order of wasps! Not that the language was only for wasps or anything, but _The more you know!_